test/bigtest/network/config.js

```diff
@@ -0,0 +1,7 @@
+// typical mirage config export
+// http://www.ember-cli-mirage.com/docs/v0.4.x/configuration/
+export default function config() {
+  this.get('/codex-instances', (schema) => {
```

Comment:
you can write it with simplified style in this case: this.get('/codex-instances');